clusterDeployer.py

```diff
@@ -136,6 +140,27 @@ def setup_all_vms(h: host.Host, vms: List[NodeConfig], iso_path: str) -> List[Fu
     return futures
 
 
+def deploy_microshift(cluster_name: str, node: NodeConfig, external_port: str, version: str) -> None:
+    lh = host.LocalHost()
+    bmc = host.bmc_from_host_name_or_ip(node.node, node.bmc_ip, node.bmc_user, node.bmc_password)
+    h = Host(node.node, bmc)
+    name_of_final_iso = 'final.iso'
+
+    microshift_iso_builder(lh, name_of_final_iso, version)
+
+    if os.path.exists(f"./{name_of_final_iso}"):
+        logger.info(f"The file {name_of_final_iso} exists.")
+    else:
+        logger.error(f"The file {name_of_final_iso} does not exist.")
+        sys.exit(-1)
+
+    nfs = NFS(lh, external_port)
+    iso = nfs.host_file(os.path.join(os.getcwd(), f"{name_of_final_iso}"))
```

Comment:
let's pass in the full path in microshfift_iso_builder, then we can have 1 var for that